### PR TITLE
switch to bpr loss

### PR DESCRIPTION
**Pull request by notthattal**
- converted the deep learning model's training from using MSE to BPR loss
- retrained the model over 900 epochs

### CODE REVIEW

**Comment by ahmed-boutar** [comment]
Thank you! It seems like we aer still having issue with the model. Like we talked about, I will look more into the eval script since the issue might be coming from that and see what the issue could be with the BPR loss function. If time permits, I will try to turn this into a hybrid rec system.
